devel/c2t/files/Makefile.patch

```diff
@@ -14,11 +14,11 @@
  
  bin/c2t: c2t.c c2t.h
 -	gcc -Wall -Wno-strict-aliasing -Wno-misleading-indentation -Wno-unused-value -Wno-unused-function -I. -O3 -o bin/c2t c2t.c -lm
-+	$(CC) -Wall -Wno-strict-aliasing -Wno-misleading-indentation -Wno-unused-value -Wno-unused-function -I. $(CFLAGS) $(LDFLAGS) -o bin/c2t c2t.c -lm
++	$(CC) -Wall -Wno-strict-aliasing -Wno-unused-value -Wno-unused-function -I. $(CFLAGS) $(LDFLAGS) -o bin/c2t c2t.c -lm
```

Comment:
The "misleading indentation" warning almost always catches real errors. I think it's not good practice to ignore real problems.

Reply:
As I have said many times, I am okay with switching everything aside of foundational ports to the current gcc.
I do not want to add a conditional patch, because nobody bothers to rebase them, and we get broken ports.

So either remove unsupported flag or blacklist Xcode gcc. I am okay with either.

Reply:
I don't see what this has to do with a conditional patch. Clearly there is a bug in the upstream code. Why are we papering over it?

Reply:
Perry, gcc-4.2 does not accept this flag. There are only two options: either drop the flag or change the compiler.

Reply:
Ah, okay. So the issue is the flag itself and not that there's a warning generated that fails?